editions/tw5.com/tiddlers/system/Sources.tid

```diff
@@ -1,10 +1,12 @@
 caption: Sources
 code-body: yes
 created: 20240313090915565
+icon: $:/core/images/transcludify
 modified: 20240313115026563
-tags: $:/tags/TiddlerInfo
+tags: $:/tags/TiddlerInfo $:/tags/MoreInfoFooter
```

Comment:
IMO `$:/tags/MoreInfoFooter` -> `$:/tags/TiddlerFooter` -- IMO the footer should be a new core feature, fist used in the docs

IMO "more" and "info" is where the functionality comes from, but in this case is redundant information and should be removed.

Reply:
Hmm, I was only thinking about the docs.  A core feature is going to require a *lot* more thinking.